content/brazilian-portuguese/java-datastructures/2d-arrays/introduction.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Introdução"
+date: 2020-02-10T13:24:17-07:00
+draft: false
+weight: 1
+---
+
+## O que é um Array 2D?
+
+Já aprendemos muito sobre vetores (arrays). Sabemos que um array pode conter muitos tipos diferentes (integers (inteiros), strings, doubles (duplos) etc.), mas que tal um array que contém arrays?
+
+![image](../img/2darray.png)
```

Comment:
Are you able to change all the alt texts of the images to a written description (in portuguese) of what the image is about? I know this is an issue with the existing workshop but for now you can focus on getting this correct here.

```suggestion
![REPLACE WITH PORTUGUESE DESCRIPTION OF 2D ARRAY IMAGE](../img/2darray.png)
```

Reply:
Done